orchestrator/api/community_skills.py

```
@@ -0,0 +1,303 @@
+
+    Returns (risk_score: int, details: list[str]).
+    """
+    # Try PluginScanService first if available
+    try:
+        from services.plugin_scan_service import PluginScanService
+
+        scanner = PluginScanService()
+        result = scanner.scan(content)
+        return result.get("risk_score", 0), result.get("details", [])
+    except (ImportError, Exception):
+        pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)